src/js/components/FormField/stories/RequiredLabel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, FormField, Form, Text, Button, Grommet } from 'grommet';
+
+const FormFieldLabel = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={
+        required ? (
+          <Box direction="row">
+            <Text>{label}</Text>
+            <Text color="red">*</Text>
+          </Box>
+        ) : (
+          label
+        )
+      }
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+const LabelFormField = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large">
+      <Form>
+        <FormFieldLabel name="firstName" label="FirstName" required />
+        <FormFieldLabel name="LastName" label="LastName" required />
+        <FormFieldLabel name="email" label="Email" />
+        <Button type="submit" label="Submit" primary />
+        <Text margin={{ left: 'small' }} size="small" color="red">
```

Comment:
I think we would want to use grommet colors here, what about `status-error`? https://v2.grommet.io/color

Reply:
Of course I can change this!